python/cuml/cluster/hdbscan/prediction.pyx

```diff
@@ -160,6 +162,9 @@ def all_points_membership_vectors(clusterer, batch_size=4096):
         cluster ``j`` is in ``membership_vectors[i, j]``.
     """
 
+    if batch_size <= 0:
+        raise ValueError("batch_size should be in integer that is > 0")
```

Comment:
There is a typo here and in addition I would suggest a slightly different wording since this not only a recommendation, but a requirement.
```suggestion
        raise ValueError("batch_size must be > 0")
```